src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,21 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
+        OrderService orderService = new OrderServiceImpl();
         User user;
         try {
-            user = authenticationService.login("bob", "1234");
+            user = authenticationService.login("bob", "1254");
+            orderService.completeOrder(user.getUserId());
```

Comment:
about this `completeOrder()`, would be better to leave it outside the try block, I'd always strive to keep the try blocks shorter, to make it more obvious where the exception might be thrown, and that's and overall good practice